pmd-java/src/main/java/net/sourceforge/pmd/lang/java/ast/ASTAnyTypeDeclaration.java

```diff
@@ -18,6 +20,18 @@
      */
     TypeKind getTypeKind();
 
+
+    /**
+     * Retrieves the body of the declaration.
```

Comment:
Hm I forgot to update that comment. This method was previously named something like "getBody" and retrieved the body of the type declaration, but that didn't solve the problem so I changed it